inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/pojo/CsvSourceInfo.java

```diff
@@ -28,17 +29,18 @@
  * CsvSourceInfo
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
+@SuperBuilder
 public class CsvSourceInfo extends SourceInfo {
 
-    private String delimiter;
-    private String escapeChar;
+    private Character delimiter;
+    private Character escapeChar;
```

Comment:
Why change the type to `Character`, what if the delimiter is not a `Character` like `\r\n`?

Reply:
There will be a new decoder to extract messages with multi-characters as one delimiter. 
The performance gap between a one-character scenario and a multi-character scenario is about 5-7 times